### PR TITLE
Bump dependencies, configure Ember for 4.0

### PR DESCRIPTION
**Pull request by mixonic**
Extracted from https://github.com/Addepar/ember-table/pull/925

### CODE REVIEW

File: config/ember-try.js [hunk 1]
```diff
@@ -13,6 +13,9 @@ module.exports = function() {
         {
           name: 'ember-lts-2.18',
           npm: {
+            dependencies: {
+              'ember-test-selectors': '5.0.0',
+            },
```
Comment (kpfefferle) [inline config/ember-try.js:18]: Is this needed if the default in `package.json` is `^5.0.0`, or does locking 2.18 to *exactly* `5.0.0` make a difference?

**Review by kpfefferle — APPROVED** [review]

File: config/ember-try.js (hunk 1 above)
Comment (mixonic) [reply config/ember-try.js:18]: It is needed exactly. For our 2.12 app at Addepar we use resolutions to pin it. I could use resolutions here as well, maybe.

Going to land this and move on though, I have a few more unblocking-other-people changes I think I can get in here quickly.

Comment (mixonic) [reply config/ember-try.js]: nm, need to rebuild anyway so trying to drop it. Will try resolutions if it is needed.

Comment (mixonic) [reply config/ember-try.js]: Well, seems like I was wrong. This must only be a limitation of 2.12! :tada:

**Pull request merged**